fire/README.md

```diff
@@ -15,27 +15,27 @@ As of this moment, the userscript posts flags _and_ MS feedback.
   ![Fire icon on report](fire.png)
 - SmokeDetector's _last_ report can be opened in a popup using <kbd>Ctrl</kbd>+<kbd>Space</kbd>
 - The popup will contain:
-  - The reported post's contents.
-    - All links in a post will show their target url when you hover your mouse over them.
-    - Code blocks in the post can be expanded by clicking on them.
-  - 5 feedback buttons.
-    - `[spam]`, `[rude]`, `[tp-]`, `[naa-]`, `[fp-]`, for spam, rude/abusive, vandalism, NAA/VLQ, false postitive.
-    - These can be selected with <kbd>1</kbd>-<kbd>5</kbd>, then "clicked" by pressing <kbd>Space</kbd> or <kbd>Enter</kbd>
-    - The buttons can contain a number in parentheses. This indicates how many users have sent that type of feedback.
   - A header for the post's site
     - Clicking this header or pressing <kbd>6</kbd> will open the reported post on the site.
   - A "Close" button.
-- When you submit `spam` feedback, the post will _also_ be flagged as "spam" on the SE network (configurable setting).
-  - Provided you [have authorized MetaSmoke](https://metasmoke.erwaysoftware.com/authentication/status) to cast flags from your account.
-- `rude` feedbacks "tpu-", just like `spam`, but without casting any flags. The number of both kinds of feedback will be displayed in `spam`, as they're equal to MetaSmoke.
+  - 5 feedback buttons.
```

Comment:
Did you mean ***6** feedback buttons*?

Reply:
Yep. I missed a few things when editing the Readme.md file. I was just too tired, I guess. I think this error was because I copy-&-pasted over the changes I'd already made, because I specifically remember changing that number.

Thanks for pointing this out.